storage/database/db_manager.go

```diff
@@ -413,7 +415,7 @@ func newDatabase(dbc *DBConfig, entryType DBEntryType) (Database, error) {
 	case MemoryDB:
 		return NewMemDB(), nil
 	case DynamoDB:
-		return NewDynamoDB(dbc.DynamoDBConfig)
+		return NewDynamoDB(dbc.DynamoDBConfig, dbc.DBName)
```

Comment:
You can use `dbBaseDirs[entryType]` instread of `dbc.DBName`.

Reply:
review reflected